tests/acceptance/requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyHamcrest>=1.8.3
+git+https://github.com/qaenablers/python-qautils.git@develop#egg=python-qautils
+python-keystoneclient==1.3.0
```

Comment:
This is a good point. So far, we are considering the content in `tests/acceptance/` as a independent *subproject* itself, so that it should manage its own requirements.